PokeAlarm/Utils.py

```diff
@@ -273,6 +273,45 @@ def get_pokemon_gender(gender):
         return u'\u26b2'  # neutral
     return '?'  # catch all
 
+# Returns the types for a pokemon
+def get_base_types(pokemon_id):
+    if not hasattr(get_base_types, 'info'):
+        get_base_types.info = {}
+        file_ = get_path('data/base_stats.json')
+        with open(file_, 'r') as f:
+            j = json.loads(f.read())
+        for id_ in j:
+            get_base_types.info[int(id_)] = {
+                "type1": j[id_].get('type1'),
+                "type2": j[id_].get('type2')
+            }
+    return get_base_types.info.get(pokemon_id)
+    
+def is_raid_boss_weather_boosted(pokemon_id, weather_id):
```

Comment:
This function could use some optimization.

Ideally we would just keep a dict of weather_id -> boosted_types and returning something like `type1 in dict or type2 in dict`. 

We should also try to strive for using types as `ints` (much faster to compare and less troubles with encoding) (I know that base_stats doesn't have this currently, but we're working on it)